Web-Scraping/ScrappingHackerNewsWebsite.py

```diff
@@ -0,0 +1,54 @@
+'''
+Scraping the first 2 pages of Hacker news website which gives lot of Tech news(as a articles)
+which has upvotes more than 100.User can just click on story link to see the article.
+'''
+
+'''
+Program uses requests module to get web data from URL and BeautifulSoup module to parse the web data
+as HTML using html parser.
+Install requests and BeautifulSoup module before executing!
```

Comment:
LGTM :+1: 
Better add README (guide) along with this inside the folder same as your `<filename>` & you're good to go!

Signed by - @Neilblaze

Reply:
Okay. I will add readme inside the folder